--- cpmd.3.15.3/Makefile	2014-08-14 20:50:09.472942095 -0700
+++ patch-files/Makefile.gnu	2014-08-14 20:50:29.742833445 -0700
@@ -12,11 +12,12 @@
 SRC  = .
 DEST = .
 BIN  = .
-FFLAGS = -c  -O2 -fcray-pointer 
-LFLAGS = -L/usr/lib64/atlas/ -llapack -lblas -lfftw 
+FFLAGS = -c  -O2 -fcray-pointer  -fno-whole-file
+LFLAGS = -LROLL_MATHPATH -lmkl_rt -lm -lpthread -LROLL_FFTWPATH -lfftw
 CFLAGS = -c -O2 -Wall
 CPP = /lib/cpp -P -C -traditional
-CPPFLAGS = -D__Linux -D__PGI -D__GNU -DFFT_FFTW -DPARALLEL -DMYRINET -DPOINTER8  
+CPPFLAGS = -D__Linux -D__PGI -D__GNU -DFFT_FFTW -DPARALLEL -DMYRINET -DPOINTER8 \
+             -DINTEL_MKL 
 NOOPT_FLAG = 
 CC = mpicc 
 FC = mpif77 
